Tests/js/views/services/assets/ez-roleserversideviewservice-tests.js

```diff
@@ -47,6 +52,63 @@ YUI.add('ez-roleserversideviewservice-tests', function (Y) {
                 "The contentDiscovered should have been added"
             );
         },
+
+        "Should not add the contentDiscovered handler on policy editing": function () {
+            var config = {};
+
+            this.service.fire('whatever:contentDiscover', {
+                config: config,
+            });
+
+            Assert.isUndefined(
+                config.contentDiscoveredHandler,
+                "The contentDiscovered should not have been added"
+            );
+        },
+    });
+
+    isDiscoveringForAssigningRoleTest = new Y.Test.Case({
+        name: "eZ Role Server Side View Service is discovering for assign role test",
+
+        setUp: function () {
+            this.service = new Y.eZ.RoleServerSideViewService();
+        },
+
+        tearDown: function () {
+            delete this.service;
+        },
+
+        "Should return false when data is not set": function () {
+            var e = {
+                config: {
+                },
+            };
+
+            Assert.isFalse(this.service._isDiscoveringForAssigningRole(e), 'The method should return false');
```

Comment:
Test should be constructed bit different I think. You shouldn't call directly private/protected methods. You should rather test if the public-available mechanisms are making correct use of them, but as far as I can see it is tested with `"Should not add the contentDiscovered handler on policy editing"` in `contentDiscoverEventTest` test case. So I guess that whole `isDiscoveringForAssigningRoleTest` test case is not needed at all - can you confirm that?